### DISCUSSION TITLE
Relations dynamically between the instances based on permalinks

### DISCUSSION THREAD

**Discussion by siwwar** [lloc/Multisite-Language-Switcher#203] [Ideas]
Hi everybody,

First of all I would like to say thanks to all the contributors on the development of this plugin, which made the Multi-Languages functionality on WordPress much easier! thanks and great job.

Due the fact that must of us using the same pattern of permalinks structure when creating the pages or posts or any other component, for example: ( http://www.domain.com/en/about = http://www.domain.com/fr/about ).

I have a small suggestion that would make the relations between pages/posts/tags etc.. much easier, if we have the ability to auto link all the instances based on permalinks with the change of the first two characters which refer to the language.

Thank you all!
Ammar

**Comment by chk613** [comment]
any update on this? I have hundreds of pages and don't want to have to this manually?

**Comment by lloc** [comment]
No development on this is planned. There are addons that can point you in the right direction. For example: https://github.com/lloc/msls-importer
